src/app/product/[id]/page.tsx

```diff
@@ -0,0 +1,68 @@
+import { Metadata } from 'next'
+import Image from 'next/image'
+import React from 'react'
+import Counter from '../../../../components/Counter'
+import pic from '../../../img/Coffee.png'
+
+async function getItemById(id: string) {
+  const response = await fetch(`http://localhost:8083/api/v1/products/${id}`)
+
+  return response.json()
+}
+
+interface Props {
```

Comment:
Why do we need this interface?

Reply:
This interface is used to define the structure of props passed to ProductPage, which will help avoid potential errors.

Reply:
What is props?

Reply:
In React, props is a way of passing information from one component to another.